src/partials/bloblang-playground.hbs

```diff
@@ -0,0 +1,238 @@
+{{{page.contents}}}
+<!-- Action buttons -->
+<div class="button-bar">
+  <button id="clear" class="button">Clear</button>
+  <button id="prettify" class="button">Format input</button>
+  <select id="sample-dropdown" class="button">
+    {{#each (obj page.attributes.bloblang-samples) }}
+    <option value="{{this.title}}">{{this.title}}</option>
+    {{/each}}
+  </select>
+</div>
+<div class="playground-container">
+  <!-- Editors -->
+  <section class="editors">
+
+    <!-- Editor containers -->
+    <div class="editor-container">
+      <!-- Input editor -->
+      <details open>
+        <summary>Input (JSON)</summary>
+        <div class="editor" id="ace-input" contenteditable="true">{ "message": "hello world" }</div>
+      </details>
+
+      <!-- Mapping editor -->
+      <details open>
+        <summary>Mapping</summary>
+        <div class="editor" id="ace-mapping" contenteditable="true">root = this</div>
+      </details>
+    </div>
+  </section>
+
+  <!-- Output -->
+  <section class="output-section">
+    <div class="editor-container">
+      <details open>
+        <summary>Output</summary>
+        <div class="editor" id="ace-output" contenteditable="true">"Output will appear here..."</div>
+      </details>
+      <details id="metadata-details">
+      <summary>Metadata output</summary>
+        <div class="editor" id="ace-metadata" contenteditable="true">"Metadata will appear here..."</div>
+      </details>
+    </div>
+  </section>
+</div>
+<script>
+let aceInputEditor, aceMappingEditor, aceOutputEditor,metadataDetails;
+
+// Keys for localStorage
+const localStorageKeys = {
+  input: "blobl-editor-input",
+  mapping: "blobl-editor-mapping",
+};
+
+// Default values for editors
+const defaultInput = '{ "message": "hello world" }';
+const defaultMapping = "root = this";
+const defaultOutput = "Output will appear here...";
+const defaultMeta = "Output metadata will appear here...";
+
+document.addEventListener("DOMContentLoaded", () => {
+  metadataDetails = document.getElementById("metadata-details");
+
+  // Initialize input editor
+  aceInputEditor = ace.edit("ace-input");
+  aceInputEditor.setTheme("ace/theme/github");
+  aceInputEditor.session.setMode("ace/mode/json");
+  aceInputEditor.session.setTabSize(4);
```

Comment:
What's 4? 4 tabs? 4px? Consider putting it into a FINAL_VARIABLE. If this doesn't make sense in this context, please disconsider.

Reply:
Spaces. Good idea